src/AddressRegistry.Api.BackOffice/Address/AddressController-Propose.cs

```diff
@@ -75,9 +78,17 @@ public partial class AddressController
                 var addressPersistentLocalId =
                     new AddressPersistentLocalId(persistentLocalIdGenerator.GenerateNextPersistentLocalId());
 
+                var postalMunicipality = await syndicationContext.PostalInfoLatestItems.FindAsync(new object[] { postalCodeId.ToString() }, cancellationToken);
+                if (postalMunicipality is null)
+                {
+                    throw new PostalCodeMunicipalityDoesNotMatchStreetNameMunicipalityException();
```

Comment:
PostalCodeDoesNotMatchAnyMunicipalityException zou ik doen, streetname is niet nodig in die naam denk ik?

Reply:
It does match a municipality. Only not the one that streetname matches.